samples/nrf_rpc/protocols_serialization/server/Kconfig

```diff
@@ -22,6 +22,20 @@ config LOG_BACKEND_RTT
 config LOG_BACKEND_UART
 	default n
 
+if SOC_NRF54L15
+
+config BT_CTLR_ECDH
+	default n
+
+config BT_USE_PSA_API
+	default y if !BT_CTLR_ECDH
+	select PSA_WANT_ALG_ECB_NO_PADDING
```

Comment:
Why do you need ECB? seems like workaround

Reply:
This is a dependency of CMAC which is enabled by BT_CRYPTO. I guess BT_CRYPTO should select it but I'm not super expert on BLE crypto so I didn't want to touch it. But I can change it and ask BLE guys for review.

Reply:
I'm pretty sure ECDH is self-sufficient by self. CMAC is used for key and salt derivation. Is it required for our sample?

Reply:
So the thing is `BT_SMP` selects `BT_CRYPTO` which selects `PSA_WANT_ALG_CMAC`. But if you try to build anything with `PSA_WANT_ALG_CMAC` but without `PSA_WANT_ALG_ECB_NO_PADDING` you get a compile error because `CMAC`.

Reply:
Isn't this Bluetooth KConfig bug? I'm pretty sure it should be fixed there.

Reply:
It should be something like this, that I've done for ble mesh: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/Kconfig#L1418-L1449